guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -14,14 +14,30 @@ To create a file type repository in a local directory, complete the following pr
 
 . Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+** For RHEL 7:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
---enable={project-client-RHEL7-url}
+--enable={RepoRHEL7ServerSatelliteUtils}
+----
+** For RHEL 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
 ----
 endif::[]
 . Install the Pulp Manifest package:
+** For RHEL 7:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install-project} tfm-pulpcore-python-pulp_manifest
+----
+** For RHEL 8:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
```

Comment:
Below the block are also instructions for RHEL 7 to install it without downtime (and with `yum install install`). Should those also be duplicated?

Also, I wonder why the instructions below disable the repo again after installing it. That looks bad.

Additionally, since it uses `foreman-maintain` and package locking only happens in Satellite (by default), the whole block should be behind `ifdef::satellite[]`.